.packit.yaml

```diff
@@ -1,18 +1,37 @@
+# See the documentation for more information:
+# https://packit.dev/docs/configuration/
+
+specfile_path: dnf5.spec
+
+# add or remove files that should be synced
 files_to_sync:
   - dnf5.spec
+  - .packit.yaml
+
+actions:
+  fix-spec-file:
+    # set the correct version for global project_version_{major,minor,patch} macros needed by %build
+    bash -c "sed -i -r \"s/^%global project_version_major (\s*)\S+/%global project_version_major \1`echo ${PACKIT_PROJECT_VERSION} | cut -d. -f3`/\" dnf5.spec"
+    bash -c "sed -i -r \"s/^%global project_version_minor (\s*)\S+/%global project_version_minor \1`echo ${PACKIT_PROJECT_VERSION} | cut -d. -f3`/\" dnf5.spec"
+    bash -c "sed -i -r \"s/^%global project_version_patch (\s*)\S+/%global project_version_patch \1`echo ${PACKIT_PROJECT_VERSION} | cut -d. -f3`/\" dnf5.spec"
+
+# name in upstream package repository or registry (e.g. in PyPI)
+upstream_package_name: dnf5
+# downstream (Fedora) RPM package name
+downstream_package_name: dnf5
 
 jobs:
   - job: propose_downstream
     trigger: release
     dist_git_branches:
-        - fedora-rawhide
-        - fedora-38
+      - fedora-rawhide
+      - fedora-38
```

Comment:
I might be missing something, but branches in fedora distgit are named without the `fedora-` prefix - `rawhide`, `f38`...

Reply:
yes, after dnf5 is in fedora 37. for now, we only want f38 and 39 I believe

Reply:
sorry I meant after dnf5 is in fedora 38, 39, 40. if we do fedora-all it will target fedora 37 I guess

Reply:
They are aliases. Maybe packit is smart enough to make a release on all fedoras in which the package is released. I would change it later though, just to be safe.